src/main/frontend/extensions/zotero.cljs

```diff
@@ -496,5 +496,5 @@
           (str "file://"
                (util/node-path.join
                 (setting/setting :zotero-linked-attachment-base-directory)
-                (str/replace-first path "attachments:" "")))]
+                path))]
```

Comment:
We need to wrap it into util function and provide unit tests

Reply:
Is it ensured that the input `path` here must be a relative path?

Reply:
> Is it ensured that the input `path` here must be a relative path?

I've reverted it.